installers/MacInstaller.hs

```diff
@@ -35,10 +42,29 @@ main = do
   copyFile "cardano-node" (dir <> "/cardano-node")
   copyFile "log-config-prod.yaml" (dir <> "/log-config-prod.yaml")
   copyFile "data/ip-dht-mappings" (dir <> "/ip-dht-mappings")
+  copyFile "data/ip-dht-mappings" (dir <> "/ip-dht-mappings")
+  copyFile "build-certificates-unix.sh" (dir <> "/build-certificates-unix.sh")
+  copyFile "tls/ca.conf" (dir <> "/ca.conf")
+  copyFile "tls/server.conf" (dir <> "/server.conf")
 
   -- Rewrite libs paths and bundle them
   _ <- chain dir $ fmap T.pack [dir <> "/cardano-launcher", dir <> "/cardano-node"]
 
+  -- Prepare postinstall script
+  createDirectoryIfMissing False resDir
```

Comment:
> The postinstall can't already be invoked -- this is installer generation time, not run time.

I don't understand. This code writes `../release/darwin-x64/Daedalus-darwin-x64/Daedalus.app/Contents/Resources/postinstall`. What invokes this script?

If this is really needed, why is `installers/data/scripts/postinstall` then still around?